Components/Entities/IFooterSection.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace GoC.WebTemplate.Components.Entities
 {
-    interface IFooterSection
+    public interface IFooterSection
```

Comment:
Not saying it is wrong, just wondering: Why visibility of this interface was changed?

Reply:
As discussed on call

Reply:
As discussed, to keep visibility in sync with other functions